Gemfile

```diff
@@ -30,6 +32,7 @@ gem 'font-awesome-sass', '~> 5.13.0'
 #used to get icons and styles
 
 gem 'devise'
+#devise helps authenticating users with its built in methods and provides helpers to get currentuser and his details taken during registrations
```

Comment:
Add this comment before defining the gem

Reply:
i have changed this